Title: Walter Báez Laguna
Date: 2019-1-25
Category: People
Tags: Stacks, RADSeq, Bio Research, Red Snappers, lutjanus, Salmon, Differential Expression, Sea Cucumber, Kallisto, transcriptome, segmentation, yanagi, splicing, differential gene-level expression

# Biography :memo:

My name is Walter Báez Laguna and I'm currently an undergraduate student at the University of Puerto Rico, Río Piedras Campus. 
I major in Computer Sciences and I'm also a former cheerleader for the UPRRP Cheerleading Team. My goal for this semester is to finsih my classes so I can graduate this year. (づ｡◕‿‿◕｡)づ 

## Hobbies :space_invader:
- I love video games! All different kinds, from MOBAs to Shooters, RPGs and MMOs, everything honestly. :video_game:
- I'm also an anime and manga fan. :octocat:
- Sleep and eating should count as hobbies. :pizza:
- Some sports are fun to do like swimming, volleyball and of course Cheerleading. :swimmer:
- Science Fiction and Fantasy books are life to me :books:
- Last, but not least spend some time browsing the internet finding new projects and interesting codes. :ghost:

# Contact Info :black_nib:
email: walter.baez1@upr.edu | alt: baez.laguna@gmail.com

github: https://github.com/Hallech

# Research Goals :microscope:

## Research 3

- Learn Kallisto and Yanagi.
- Be able to find and eleminate the false positives. 
- Try to implement the same concept Yanagi does with our own methods.
- Use De Brujin Graph and how Titus uses them for the benefit of our research.

-------------------------------------------------------------------------------------------
## Research 2

- Have a nice group dynamic and work everything well.
- Actually proccess some useful data.
- Learn Salmon and how to use edgeR.

-------------------------------------------------------------------------------------------
## Research 1

- Learn more about Stack and how to use it properly.
- Understand more about RADSeq data.
- Have a successful outcome out of the research. 
- Produce a relevant denovo_map to conclude the genomic comparison. 

-------------------------------------------------------------------------------------------

# Research Description :open_file_folder:

## Research 3

- Awaiting Design.

-------------------------------------------------------------------------------------------
## Research 2

- I'm working on a new a project with David Ortiz focused on differential expression. 
- We are going to be using the tools Salmon and edgeR for this project.
- The data we are working on is from the Sea Cucumber.

--------------------------------------------------------------------------------------------
## Research 1

- Use Stack to analyze RADSeq data to find SNPs across the Species Red Snappers.
- Conlude if lutjanus campechanus or purpureus are different species or the same one.
- Make Stack's pipeline work for once.

--------------------------------------------------------------------------------------------

# Weekly Updates :date:

## Research 3 (Spring 2019):

### Week 9 & 10 (March 10, 2019 to March 23, 2019) :clock1:
- I'm looking at the pipeline Titus used for constructing De Brujin Graphs.
- With this we want to construct the graphs and see how we can eliminate false positives.
- The first things I'm going to do is setting up the enviroment and getting familiar with
  the process before going to the next step, which is finding how to delete these false
  positives. 

### Week 7 & 8 (February 24, 2019 to March 9) :clock1:
- During this time I was working on the presentation of the Yanagi and Kalisto paper.
- I did a PowerPoint explaining the most important details and during our Lab Meeting
  we discussed it together and worked our way throught the pipeline.
- After the presentation was done I discussed with my mentor what's the next step.

### Week 5 & 6 (February 10, 2019 to February 23, 2019) :clock1:
- After reading the papers I have a bit of more knowledge and the idea of how to start working
  with the research. Yanagi is a very extensive paper, I still have some doubts on how it works
   exactly.
- I need to have a meeting with my mentor to design the work I'll be doing in the next few weeks.
- It's been a while since I worked in a bioinformatics research so I'm a bit lost again, I remember
  lots of things but some concepts are a bit confusing. 

### Week 3 & 4 (January 26, 2019 to February 9, 2019) :clock1:
- During this time I got sick so I got behind in my classes schedule. I had a lot of projects
  and homework to turn in so I did not get much time to work in the research. 
- I have the two papers I'm going to start reading in order to design the project I'll be working 
  on.
- Next week I'll sit with my mentor and discuss how to start and what is the approach of the new
  research, of course after reading the papers that will give me the background. 
  
### Week 1 & 2 (January 14, 2019 to January 25, 2019) :clock1:
- After a while out of the research enviroment I'm coming back again to research how to eleminate
  false positive after running a sequence of differential expressions using kmers. 
- For the upcoming weeks I'll be reading the two papers assigned to this topics (Kallisto and the 
  paper by Hector Corrada).
- Once the readings are done the next step will be trying to see how to approach the problem using 
  the techniques my lab is using.

--------------------------------------------------------------------------------------------

## Reserch 2 (Spring  2017):

### Week 7 & 8 (March 10, 2017 to March 22, 2017) :clock1:
- I was told to work on a script for Salmon. It has to build the index so we can run the matching progess
  afterwards. 
- I also added the matching script right after the index one to speed up the proccess. 

### Week 5 & 6 (February 24, 2017 to March 10, 2017) :clock1:
- My partner was working with the data on Trinity while I was waiting for whatever he needed me to work on.
- I was told to search the web for a way to make an enviroment available to everyone with the kmer tools and
  any other tools we would need. 

### Week 3 & 4 (February 10, 2017 to Febraury 24, 2017) :clock1:
- During these weeks I tried to do a full pipeline to run all the steps on Titus workshop from one
  bash script.
- We needed all the kmer tools installed in an enviroment and my partnert was working on that.

### Week 1 & 2 (January 30, 2017 to February 10, 2017) :clock1:
- I'm going to begin working on a new a project with David Ortiz focused on differential expression. 
- We are going to be using the tools Salmon and edgeR for this project.
- The first thing we did was go through the Titus Quantification and Differential Expression of RNAseq
  workshop: http://2016-aug-nonmodel-rnaseq.readthedocs.io/en/latest/quantification.html
  
--------------------------------------------------------------------------------------------

## Research 1 (Fall 2016):

### Week 16 (December 9, 2016 to December 16, 2016) :clock1:
- The VM is still not set up and we are stuck once again.

### Week 14 & 15 (November 25th, 2016 to December 9, 2016) :clock1:
- It took some time to get the approval, but we got it!
- Sad part is we couldn't set up the VM, it's giving us some issues since we are making
  with this new recipe to have more memory.

### Week 12 & 13 (November 11, 2016 to November 25, 2016) :clock1:
- Our options started to be less each time and we came up with another idea.
- Build a better bigger VM in Makaria (where I was previously working on) this
  way I could work in it without the administrator dilema and have the help,  
  memory and speed I need.

### Week 10 & 11 (October 28, 2016 to November 11, 2016) :clock1:
- The VM was set up, we uploaded a few files and we ran the pipeline to see if the denovo_map
  would work or if we could spot the problems.
- It took forever, literally it was running for like 3 weeks and it only did about 5 out of 20.

### Week 8 & 9 (October 14, 2016 to October 28, 2016) :clock1:
- I decided to work in the lab with my professor where I could get more advice and help
  since I the administrator settings and limited help was beginning to be another issue.
- The strategy was to set up a virtual machine so we can run a small test and view how 
  does Stack work and figure out the problems.

### Week 6 & 7 (September 30th, 2016 to October 14, 2016) :clock1:
- We started to have a lot of problems with the database for the output of the pipeline.
- Administration problems, failed uploads, repetitive data, missing fails.
- The error could not be easily fixed so we went and talked with a few peopel about it.

### Week 4 & 5 (September 16, 2016 to September 30th, 2016) :clock1:
- The time was spent trying to understand the Stack pipeline to run the entire program.
- We ran a few commands using the manual until there was no more further information for 
  denovo map logs without a reference genome. 
- Everything was put together so we can start our first denovo_map and it was left running.

### Week 2 & 3 (September 2nd, 2016 to September 16, 2016) :clock1:
- We started working with the data that arrivied, 160 GB of raw data.
- First steps were using fastqc to review the data, trimming and cleaning it.

### Week 1 (August 21ft, 2016 to September 2nd, 2016) :clock1:
- Arrieved at the lab, met the professors and received my instructions.
- Read the Stack presentation and manual given to me at the lab.
- Waiting on the RADSeq data that hasn't arrived and on more instructions.
- There was no one on the lab (Sept. 2nd) because they went fishing for more research individuals 
  so I went and did my hours creating my GitHub and setting up my Weekly Reports.
